src/task_scl_eff_pot_hab.py

```diff
@@ -131,12 +131,33 @@ def calc(self):
 
         hii_ecoregion = self.ecoregion_country.add(hii_sum)
 
-        min_patch_size = self.density.remap(
-            self.remap_lists["density"], self.remap_lists["min_patch_size"]
-        ).updateMask(hii_ecoregion)
-        min_stepping_stone_size = self.density.remap(
-            self.remap_lists["density"], self.remap_lists["min_stepping_stone_size"]
-        ).updateMask(hii_ecoregion)
+        def density_to_patch_size(ft):
+            med_density_eco = ee.Number(ft.get('MED_DENSITY_ECO'))
+            med_density_biome = ee.Number(ft.get('MED_DENSITY_ECO'))
+            med_density_eco_biome = ee.Number(ft.get('MED_DENSITY_ECO'))
+            density_val = ee.Algorithms.If(med_density_eco.gt(0),
+                med_density_eco,
+                ee.Algorithms.If(med_density_biome.gt(0),
+                    med_density_biome,
+                    ee.Algorithms.If(med_density_eco_biome.gt(0),
+                        med_density_eco_biome,
+                        1)))
+            min_core_size = ee.Number(500).divide(density_val).int()
```

Comment:
Should abstract hardcoded values into class properties

Reply:
will update this